icu4j/main/classes/core/src/com/ibm/icu/impl/units/MeasureUnitImpl.java

```diff
@@ -2,29 +2,18 @@
 // License & terms of use: http://www.unicode.org/copyright.html
 package com.ibm.icu.impl.units;
 
-import java.math.BigDecimal;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-
 import com.ibm.icu.util.BytesTrie;
 import com.ibm.icu.util.CharsTrie;
 import com.ibm.icu.util.CharsTrieBuilder;
 import com.ibm.icu.util.ICUCloneNotSupportedException;
 import com.ibm.icu.util.MeasureUnit;
 import com.ibm.icu.util.StringTrieBuilder;
 
-public class MeasureUnitImpl {
-
-    /**
-     * The full unit identifier.  Null if not computed.
-     */
-    private String identifier = null;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Comparator;
```

Comment:
Move these imports above the "import com.*" imports. That's how Eclipse is configured to save them.

(For the future: figure out how to make IntelliJ get this right, to not end up in an edit war with Eclipse.)

Reply:
Yoshito's comment in https://github.com/unicode-org/icu/pull/1374 :

```
@younies It looks files touched by you uses different import statement ordering. We set up our preferences in Eclipse project files. Could you please honor the setting? Easiest way to handle this is to import ICU projects to Eclipse, and let Eclipse to do the work.
```

I've seen I can change the import order in clang-format too.

Reply:
Looking into Eclipse, the desired order of imports is: ['java', 'javax', 'org', 'com'].

Reply:
My commit changed the import order back, so this is done.